_drafts/oryx/browser-compatibility.md

```diff
@@ -0,0 +1,24 @@
+# Supported browsers / viewports
+Oryx supports and is being tested on all modern browsers and operating systems:
+
+* iOS 16.x (Safari latest, Google Chrome latest)
+* Android 13.x (Google Chrome latest)
+* macOS Ventura (Safari latest, Google Chrome latest) 
+* Windows 10 and higher (Google Chrome latest, Edge latest)
+
+This allows us to cover more than 95% of operating systems and 93% of browsers used worldwide.
```

Comment:
Would we change this numbers every time? And are we sure in these numbers - I think they are really depends on the platform where stats are gathered. I'd not put them.

Reply:
agree, removed that completely